lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -123,7 +120,7 @@ def air_loop_hvac_apply_prm_baseline_fan_power(air_loop_hvac)
        system_type == 'SZ_CV'
 
       # Calculate the allowable fan motor bhp for the air loop
-      allowable_fan_bhp = air_loop_hvac_allowable_system_brake_horsepower(air_loop_hvac) + fan_pwr_adjustment_bhp
+      allowable_fan_bhp = air_loop_hvac_allowable_system_brake_horsepower(air_loop_hvac)
```

Comment:
`fan_pwr_adjustment_bhp` were double counted. The `air_loop_hvac_fan_power_limitation_pressure_drop_adjustment_brake_horsepower()` is called within `air_loop_hvac_allowable_system_brake_horsepower()`.